executor/hnswlib_searcher.py

```diff
@@ -32,13 +32,14 @@ def __init__(
         metric: str = DEFAULT_METRIC,
         dim: int = 0,
         max_elements: int = 1_000_000,
-        ef_construction: int = 400,
+        ef_construction: int = 200,
         ef_query: int = 50,
-        max_connection: int = 64,
+        max_connection: int = 16,
         dump_path: Optional[str] = None,
         traversal_paths: Iterable[str] = ('r',),
```

Comment:
```suggestion
        traversal_paths: str = 'r',
```

Reply:
Why? This is the standard design we've used in all Executors, no? 

[here](https://github.com/jina-ai/executors/blob/eb48ea13f4305652a0863dfadb619c34c22c0045/jinahub/encoders/audio/AudioCLIPEncoder/executor/audio_clip_encoder.py#L25)
[here](https://github.com/jina-ai/executors/blob/f576370347c28d383af8ebc516efbc2adcdecbf0/jinahub/indexers/storage/PostgreSQLStorage/postgres_indexer.py#L26)

Reply:
https://docs.jina.ai/fundamentals/document/documentarray/#traverse-nested-structure

Reply:
No, because that's for only one traversal. Executors use `traversal_path**s**`, meaning it can be several traversals. Yes, by default it's only `['r']`, which is one traversal.

Reply:
Nvm, I see this change is indeed breaking. I will address it in another PR